tests/components/input_boolean/test_init.py

```diff
@@ -33,26 +35,30 @@
 
 async def test_methods(hass):
     """Test is_on, turn_on, turn_off methods."""
-    assert await async_setup_component(hass, DOMAIN, {DOMAIN: {"test_1": None}})
+    assert await async_setup_component(hass, DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4751571)